app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+
+    if len(command_parts) != 3:
+        return
```

Comment:
better to raise some message 
To make the user understand that something went wrong.

```suggestion
        raise ...
```

Reply:
not fixed